### DISCUSSION TITLE
Solve did not converge

### DISCUSSION THREAD

**Discussion by Salma-Mao** [idaholab/moose#20394] [Q&A General] [answered]
Dear everyone,

I am new to MOOSE. When I am trying  to compile the project  [GrandPotentialPlanarGrowth.i](https://github.com/idaholab/moose/blob/next/modules/phase_field/examples/anisotropic_interfaces/GrandPotentialPlanarGrowth.i), I find that the example does not converge. What can I do to resolve this problem. 

![image](https://user-images.githubusercontent.com/54804946/155545462-fb4505d2-414a-4b37-b1f2-95a531c76511.png)

Thanks,
MS

**Comment by ykvishal** [comment]
I could run this simulation.  
![image](https://user-images.githubusercontent.com/31565781/155582422-ae708a5d-3752-412f-bb72-f723a048e117.png)

**Comment by GiudGiud** [comment, marked as answer]
@shuine-09 
Please see this page to diagnose convergence issues. 
https://mooseframework.inl.gov/application_usage/failed_solves.html

@ykvishal and @shuine-09 anything particular in how you ran the case (#mpi, threads?, old MOOSE version or new one?) ?

**Reply by ykvishal** [reply]
In my case, I installed MOOSE 2 weeks ago. Also I could run this input file with single core, 2 cores and 12 cores. It worked all the time. Although, the number of time-steps were different in all cases.

**Reply by Salma-Mao** [reply]
I run this simulation again.

![image](https://user-images.githubusercontent.com/54804946/155644448-192cf4c0-cdb1-4f92-a150-e62862fe88f1.png)

 However, When  output files are opened   by ParaView 5.10.0, I get the Error. Did the Error mean that the solve did not converge? 

![image](https://user-images.githubusercontent.com/54804946/155644942-27818832-08a2-4585-9474-f24ede95cd31.png)

**Reply by GiudGiud** [reply]
That version of paraview isn’t compatible with moose because we name side sets and nodesets the same for the most part. 

Please either downgrade or upgrade

**Reply by Salma-Mao** [reply]
The version of ParaView have been changed. And the output has been opened successfully. Thanks for your help.
